modules/gapi/test/gapi_desc_tests.cpp

```diff
@@ -67,16 +69,18 @@ TEST(GAPI_MetaDesc, VecOwnMatDesc)
     cv::gapi::own::Mat(240, 320, CV_8U, nullptr),
     cv::gapi::own::Mat(480, 640, CV_8UC3, nullptr)};
 
-    const auto desc = cv::gapi::own::descr_of(vec);
-    EXPECT_EQ(CV_8U, desc[0].depth);
-    EXPECT_EQ(1,       desc[0].chan);
-    EXPECT_EQ(320,     desc[0].size.width);
-    EXPECT_EQ(240,     desc[0].size.height);
-
-    EXPECT_EQ(CV_8U, desc[1].depth);
-    EXPECT_EQ(3,       desc[1].chan);
-    EXPECT_EQ(640,     desc[1].size.width);
-    EXPECT_EQ(480,     desc[1].size.height);
+    const auto desc = descr_of(vec);
```

Comment:
Idea was not change existing tests to test ADL but to add new ones, that test it for all (or at least new one) versions

Reply:
So is it resolved or not?